### DISCUSSION TITLE
XXXX 1 pin not visited, fall back to feedthrough mode

### DISCUSSION THREAD

**Discussion by faisaladilquadery** [The-OpenROAD-Project/OpenROAD-flow-scripts#1567] [Q&A]
I have been facing these warnings for some designs I synthesized with genus and implemented in OpenRoad. These warnings are showing up during detailed routing. What to these warnings actually indicate ?

![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/140678965/a79c20f4-faf1-4129-b4ff-c8a0c233158c)

**Comment by maliberty** [comment]
It means the route guides from global routing don't cover all the pins of the net and drt is trying to fix them up.  A test case would be helpful if you can provide one (in which case open an issue).

**Reply by faisaladilquadery** [reply]
Hi @maliberty thanks for the reply. 
Can you please guide me in how to provide a test case, then I will open a Issue.
and can u deduce any particular reason global routing isnt covering all the pins and nets ?

**Comment by maliberty** [comment]
If you are using ORFS then 'make global_route_issue' should package a test case.

**Reply by gkamendje** [reply]
I am seeing the same issue `[WARNING DRT-0225] clknet_1_0__leaf_i_sys_clk 6 pin not visited, fall back to feedthrough mode.`. Here is a test case that can be used to reproduce. https://drive.google.com/file/d/11OVLyyswdMCozIKECvxi08zB2f4SSwNG/view?usp=share_link

**Reply by maliberty** [reply]
@eder-matheus please take a look
